var/spack/repos/builtin/packages/gcc/package.py

```diff
@@ -73,6 +73,9 @@ class Gcc(AutotoolsPackage):
     variant('piclibs',
             default=False,
             description='Build PIC versions of libgfortran.a and libstdc++.a')
+    variant('strip',
+            default=False,
+            description='Strip executables to greatly reduce installation size')
```

Comment:
This line is one character too long:
```
var/spack/repos/builtin/packages/gcc/package.py:78: [E501] line too long (80 > 79 characters)
```
If you put the entire `variant()` directive on a single line, it will be ignored by flake8. You could also drop the "greatly" to make it fit.